### PR TITLE
fix h_caption_loqs_label

### PR DESCRIPTION
**Pull request by nikolas-burkoff**
FYI @npaszty 

So this fixes the breaking example :

```
library(scda)
ADLB <- synthetic_cdisc_data("latest")$adlb
caption_label <- goshawk:::h_caption_loqs_label(loqs_data = ADLB)
``` 

So  the output of `loq_values <- h_identify_loq_values(loqs_data)` seems to be a data.frame and if there's more than one row the check `if (is.na(loq_values$LLOQC)) {` gives an error so I'm now looking entry by entry to convert NA to "NA".

### CODE REVIEW

**Comment by nikolas-burkoff** [comment]
> sorry, what issue is this related to.

Sorry no issue, it's just this function fails with the upgrade of R versions - i.e. if you are on R 4.2.1 and try the example above R gives an error - this PR is to fix it.

**Comment by npaszty** [comment]
@nikolas-burkoff 
okay got it.
do you need my approval? I would have to test this in an R 4.2.1 version context and I guess that means pulling a docker image etc.?

**Comment by nikolas-burkoff** [comment]
No need for your approval @npaszty the core team will review but as you know the code probably as well as any of us, you may know of some downstream problem of changing this code so just wanted to draw your attention to the PRs existence :)

**Comment by npaszty** [comment]
@nikolas-burkoff 
I think you're giving me too much credit on knowing the code. ;-)

I believe the application of "NA" in the conditional chunk is to ensure that the LOQ footnote displays "NA" rather than a blank.

File: R/utils.R
Comment (nikolas-burkoff) [inline R/utils.R:88]: This will convert the LLOQC, ULOQC values from numbers to characters which I assume is what happened previously without ill effects

Comment (npaszty) [reply R/utils.R]: okay. sounds good. thanks.

**Review by mhallal1 — APPROVED** [review]

**Pull request merged**
